main.py

```diff
@@ -120,16 +133,12 @@ def registration():
 
         with open("access_token.txt") as f:
             if access_token != f.read():
-                print("bad token")
+                logging.INFO("bad token")
                 flash("Incorrect access token", "error")
                 return render_template("registration.html")
 
-        if "@" not in email:
-            flash("Incorrect email", "error")
-            return render_template("registration.html")
-
-        if "." not in email:
-            flash("Incorrect email", "error")
+        if check_email(email) is False:
```

Comment:
Much better! Though, I think the `not` keyword would be more idiomatic. i.e. `if not some_condition`